pulsar-client/src/main/java/org/apache/pulsar/client/api/Message.java

```diff
@@ -73,9 +73,26 @@
      */
     MessageId getMessageId();
 
+    /**
```

Comment:
Java changes in C++ Client PR?

Reply:
Alternately just merge the JAva PR and the difference won't be seen in this PR.

Reply:
@jai1 this change is based on #734. and they are organized in two different commits. once you merge #734 , this change will be only c++ changes.

Reply:
rebased.